src/usecases/retrieveAverageParticipantsInVisit.js

```diff
@@ -0,0 +1,23 @@
+const retrieveAverageParticipantsInVisit = ({ getDb }) => async (trustId) => {
+  if (!trustId) return { error: "A trustId must be provided." };
+
+  const db = await getDb();
+  const [{ average_participants: averageParticipantsInVisit }] = await db.any(
+    `SELECT AVG(participants) as average_participants
```

Comment:
What is returns if there is a scheduled visit with 3 participants and another visit with 0 (e.g. visit hasn't occurred yet or was cancelled)? Does it return 3 or 1.5 as the average?

Reply:
Good question! I'll write a test for this and update so it returns 3.

Reply:
Just checked it returns 3! We don't have this problem here because there won't be any events for that visit so no events to group for that visit.